webapp/src/App.js

```diff
@@ -175,7 +175,7 @@ function AppRoutes() {
           )}
         />
         <Route
-          path="/offering/:id"
+          path="/offerings/:offeringId/products"
```

Comment:
Make this `offering/:id`. The singular is more natural for user-facing language.